lib/assets/javascripts/utils/collateTable.js

```diff
@@ -0,0 +1,31 @@
+/* 
+  tablesorter is an external library located in vendor/tablesorter.
+
+  it does not provide us with icons though, so we add our own below along
+  with logic to change them to up/down arrows when the user sorts the column
+*/
+
+(function(ctx){
+  var swapIcons = (function(el){
+    if($(el).find("span.fa").hasClass('fa-sort') || $(el).find("span.fa").hasClass('fa-sort-asc')){
+      $(el).find("span.fa").removeClass('fa-sort').removeClass('fa-sort-asc').addClass('fa-sort-desc');
+    }else{
+      $(el).find("span.fa").removeClass('fa-sort-desc').addClass('fa-sort-asc');
+    }
+  });
+  
+  ctx.init = ctx.init || (function(options){
+    if($ && options && options.selector){
+      /* Bind the table to the external tablesorter JS (see vendor/tablesorter) */
+      $(options.selector).tablesorter(); 
+    
+      /* Tablesorter doesn't provide icons so we add our own here */
+      /* we also bind their click event to display the up/down arrow */
+      $(options.selector + " th.tablesorter-headerUnSorted:not(.sorter-false) div.tablesorter-header-inner")
```

Comment:
I would simplify this selector, so that it does not depend on tablesorter to insert the span. (e.g. $(options.selector + 'th') and at the end invoke $(options.selector).tablesorter(); to augmentate the HTML according to that library. Note we would need to excludethe actions columns but there is already a class 'sorter-false'.